doc/libpmempool/pmempool_feature_query.3.md

```diff
@@ -56,11 +56,11 @@ _UW(pmempool_feature_disable) - toggle or query pool set features
 ```c
 #include <libpmempool.h>
 
-_UWFUNCR1(int, pmempool_feature_query, *path, enum pmempool_feature feature, unsigned flags)
+_UWFUNCR12(int, pmempool_feature_query, *path, enum pmempool_feature feature, unsigned flags)
```

Comment:
This is not the correct fix. _UWFUNCR12 expects two string arguments.
The problem is with how we use _UWFUNCR1 here - have a look at its documentation.